vlasiator.cpp

```diff
@@ -1360,6 +1391,11 @@ int main(int argn,char* args[]) {
       double timePerSecond=double(after  - startTime) / (P::t-P::t_min+DT_EPSILON);
       logFile << "(MAIN): All timesteps calculated." << endl;
       logFile << "\t (TIME) total run time " << after - startTime << " s, total simulated time " << P::t -P::t_min<< " s" << endl;
+      logFile << "\t (TIME) total node-hours " << nNodes*(after - startTime)/3600 << " h" << endl;
+      #if _OPENMP
+         logFile << "\t (TIME) total core-hours " << omp_get_max_threads()*mpiProcs*(after - startTime)/3600 << " h" << endl;
```

Comment:
And I would suggest to reorder and have all of these (above and here) say sth like "(TIME) total x.y node-hours" instead of "node-hours x.y h"